docs/content/documentation/content/page.md

```diff
@@ -6,98 +6,98 @@ weight = 30
+create a page at `[base_url]/about`). (Note the lack of an underscore; if the file
 were named `_index.md`, then it would create a **section** at `[base_url]/about`, as
-discussed in the prior part of this documentation.  But naming the file `index.md` will
+discussed in a previous part of this documentation.  In contrast, naming the file `index.md` will
 create a **page** at `[base_url]/about`).
 
 If the file is given any name *other* than `index.md` or `_index.md`, then it will
-create a page with that name (without the `.md`). So naming a file in the root of your
-content directory `about.md` would also create a page at `[base_url]/about`.
-Another exception to that rule is that a filename starting with a datetime (YYYY-mm-dd or [a RFC3339 datetime](https://www.ietf.org/rfc/rfc3339.txt)) followed by
+create a page with that name (without the `.md`). For example, naming a file in the root of your
+content directory `about.md` would create a page at `[base_url]/about`.
+Another exception to this rule is that a filename starting with a datetime (YYYY-mm-dd or [an RFC3339 datetime](https://www.ietf.org/rfc/rfc3339.txt)) followed by
 an underscore (`_`) or a dash (`-`) will use that date as the page date, unless already set
-in the front-matter. The page name will be anything after `_`/`-` so a filename like `2018-10-10-hello-world.md` will
+in the front matter. The page name will be anything after `_`/`-`, so the file `2018-10-10-hello-world.md` will
 be available at `[base_url]/hello-world`. Note that the full RFC3339 datetime contains colons, which is not a valid
 character in a filename on Windows.
 
-As you can see, creating an `about.md` file is exactly equivalent to creating an
+As you can see, creating an `about.md` file is equivalent to creating an
 `about/index.md` file.  The only difference between the two methods is that creating
-the `about` folder allows you to use asset colocation, as discussed in the
-[Overview](@/documentation/content/overview.md#assets-colocation) section of this documentation.
+the `about` directory allows you to use asset co-location, as discussed in the
+[overview](@/documentation/content/overview.md#asset-co-location) section.
 
-## Front-matter
+## Front matter
 
-The front-matter is a set of metadata embedded in a file. In Zola,
-it is at the beginning of the file, surrounded by `+++` and uses TOML.
+The TOML front matter is a set of metadata embedded in a file at the beginning of the file enclosed
+by triple pluses (`+++`).
 
-While none of the front-matter variables are mandatory, the opening and closing `+++` are required.
+Although none of the front matter variables are mandatory, the opening and closing `+++` are required.
 
-Here is an example page with all the variables available.  The values provided below are the default
-values.
+Here is an example page with all the available variables. The values provided below are the
+default values.
 
-```toml
+```TOML
```

Comment:
those are markers for the syntax highlighter, I don't think we want to change them as they usually work by matching the file extension.

Reply:
Sorry, I didn't notice that "toml" will not show up in the output. Please revert.